201906_noise_no_lowell/noise.toml

```diff
@@ -0,0 +1,20 @@
+
+tag = "noise"
+
+[ other_components ]
+
+    [ other_components.noise ]
+
+    class = "mapsims.SONoiseSimulator"
+    return_uK_CMB = true
+    sensitivity_mode = "baseline"
+    apply_beam_correction = false
+    apply_kludge_correction = true
+    scanning_strategy = "classical"
+    no_power_below_ell = 30
```

Comment:
@thibautlouis see new parameter to cut low ell power, in this case ell = 30 is non-zero